frontend/src/Components/Suggestions.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { A } from "hookrouter";
+
+export default function Suggestions({ user }) {
+  return (
+    <div className="sticky top-0 mt-4 px-2 ">
+      <div className="flex justify-between">
+        <div className="flex gap-x-1 sm:gap-x-2 items-center">
+          <A href={`/user/${user.user_id}`}>
+            <img
+              className="rounded-full h-8 w-8 shadow"
+              src={user.profile_picture}
+              alt="Profile_Picture"
+            />
+          </A>
+          <A href={`/user/${user.user_id}`}>
+            <div className="text-sm font-medium text-gray-900">{user.name}</div>
+          </A>
+          <div className="flex gap-x-1 items-center bg-green-50 text-green-700 p-.5 px-1 font-semibold text-xs">
+            <span>
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="10"
+                fill="currentColor"
+                class="bi bi-image-fill"
+                viewBox="0 0 16 16"
+              >
+                <path d="M.002 3a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v10a2 2 0 0 1-2 2h-12a2 2 0 0 1-2-2V3zm1 9v1a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1V9.5l-3.777-1.947a.5.5 0 0 0-.577.093l-3.71 3.71-2.66-1.772a.5.5 0 0 0-.63.062L1.002 12zm5-6.5a1.5 1.5 0 1 0-3 0 1.5 1.5 0 0 0 3 0z" />
+              </svg>
+            </span>
+            5 Posts
```

Comment:
Can you add one more commit which can return the length of the posts in the lightweight call? (in the backend? - Let's do this right)

Reply:
As discussed, removing this div